### PR TITLE
Add book transitions for book list

### PR DESCRIPTION
**Pull request by JuanGutierrez43**
## Summary 

In this feature I created transitions for make a hover to the books. When you pass over a book it had to scale and have a transition. For this I used transform and transitions with scale, ease-out and in.

**New features**

- Add transitions over the books on the book list.
- Set the hover with transform and transitions with scale

## Screenshots 
Peek 2020-04-13 13-57

## Cards

https://trello.com/c/69ejgU7d/35-transitions

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -64,6 +64,13 @@ $lower-screen-width: calc(#{$book-badge-width} * 0.7);
   height: 300px;
   margin: 15px;
   width: 200px;
+  transition: 0.2s ease-out;
```
Comment (eledwinn) [inline scss/detail.scss]: Please add transition property, like `transition: transform 0.2s ease-out;`

File: scss/detail.scss [hunk 2]
```diff
@@ -64,6 +64,13 @@ $lower-screen-width: calc(#{$book-badge-width} * 0.7);
...
+
+
```
Comment (eledwinn) [inline scss/detail.scss:68]: Remove one empty line

File: scss/detail.scss [hunk 3]
```diff
@@ -64,6 +64,13 @@ $lower-screen-width: calc(#{$book-badge-width} * 0.7);
...
+  &:hover {
+    transform: scale(1.1); 
+    transition: 0.2s ease-in;
```
Comment (eledwinn) [inline scss/detail.scss]: Avoid rewrite invariant values. Try change only `transition-timing-function` property

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 1 above)
Comment (CrisVega08) [inline scss/detail.scss]: Also sort properties

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 2 above)
Comment (JuanGutierrez43) [reply scss/detail.scss:68]: I dont see this empty line. In the 69 is the `&hover` :thinking: 

I made the commit, lets check this out now!

Comment (eledwinn) [reply scss/detail.scss:68]: Perfect

**Review by eledwinn — APPROVED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**